.markdownlint.json

```diff
@@ -1,9 +1,7 @@
 {
     "default": true,
     "MD013": false,
-    "MD024": {
-        "allow_different_nesting": true
-    },
+    "MD024": false,
```

Comment:
/cc @nschonni, this reverts your change from #2780. The problem is when trying to have multiple tab-sets under the same heading with the same titles (e.g. FluentAPI/Data Annotations).

Reply:
Dotnet/docs uses individual `<!-- markdownlint disable MD024-->` statements when the old tab format causes issues instead of turning off rules entirely

Reply:
Is there an alternative new tab format the we can use instead, that doesn't have this problem?

If not, forcing these linter suppression in the doc codes seems to me more annoying/ugly than the value that MD024 brings, though that's subjective...

Reply:
I think there was supposed to be a `:::` version at some point awhile ago, but I think most of the docs for that type of thing is in the https://review.docs.microsoft.com/ site that isn't available to the public